tests/dualtor/test_standby_tor_upstream_mux_toggle.py

```diff
@@ -79,7 +85,12 @@ def test_standby_tor_upstream_mux_toggle(
                             server_ip=ip['server_ipv4'].split('/')[0],
                             pkt_num=PKT_NUM,
                             drop=True)
-    crm_facts1 = rand_selected_dut.get_crm_facts()
     unmatched_crm_facts = compare_crm_facts(crm_facts0, crm_facts1)
-    pt_assert(len(unmatched_crm_facts) == 0, 'Unmatched CRM facts: {}'
+
+    # For Cisco-8000 devices, hardware counters are statistical-based with +/- 1 entry tolerance.
+    # Hence, the available counters may not increase as per initial value for multiple facts collected.
+    logging.debug(
```

Comment:
the comments is not aligned with the code

Reply:
As there are multiple different crm facts verified like below:
ipv6_nexthop
ipv4_neighbor
acl_table
mpls_nexthop
nexthop_group_member

These can fail because of +/- 1 entry tolerance. which we are not considering as failures. This check will fail as 5 different facts conclude it. If some facts have mismatch more then +/- 1 entry will fail inside compare_crm_facts function itself. And if multiple facts have +/- 1 entry can be ignored here as we are ignoring all places.

Hence tried in multiple tests 5 was the conclusive number which is minimum required.